api/nodes/serializers.py

```diff
@@ -7,12 +7,15 @@
 
 class NodeSerializer(JSONAPISerializer):
 
+    filterable_fields = ['title', 'description']
```

Comment:
Makes sense. Worth using a frozenset here, while I'm at it? I don't expect any practical changes, per se, but it does seem the proper choice.